test/src/device-request-prompt.spec.ts

```diff
@@ -60,4 +59,46 @@ describe('device request prompt', function () {
       })
     ).rejects.toThrow(TimeoutError);
   });
+
+  it('can be opened', async function () {
+    const {page, httpsServer} = state;
+
+    await page.goto(httpsServer.PREFIX + '/device-request.html');
+
+    await expect(
+      Promise.all([
+        page.waitForDevicePrompt({timeout: 100}),
+        page.click('#bluetooth'),
+      ])
+    ).resolves.toBeTruthy();
+  });
+
+  it('can be cancelled', async function () {
+    const {page, httpsServer} = state;
+
+    await page.goto(httpsServer.PREFIX + '/device-request.html');
+
+    const [devicePrompt] = await Promise.all([
+      page.waitForDevicePrompt({timeout: 100}),
+      page.click('#bluetooth'),
+    ]);
+    expect(devicePrompt).toBeTruthy();
+    await expect(devicePrompt.cancel()).resolves.not.toThrow();
+  });
+
+  describe('waitForDevice', function () {
+    it('does not crash', async function () {
+      const {page, httpsServer} = state;
+
+      await page.goto(httpsServer.PREFIX + '/device-request.html');
+
+      const [devicePrompt] = await Promise.all([
+        page.waitForDevicePrompt({timeout: 100}),
+        page.click('#bluetooth'),
+      ]);
+      await expect(
+        devicePrompt.waitForDevice(() => false, {timeout: 10})
```

Comment:
The formatted complains for this line. Running `npm run format` should auto fix this issue.

Reply:
done in https://github.com/puppeteer/puppeteer/pull/11185/commits/bf248c21304eedb1355e8a77b6436163b15862c5